drivers/media/i2c/ov5647.c

```diff
@@ -613,65 +618,124 @@ static struct ov5647_mode supported_modes_10bit[] = {
 	 * MODE 0: 2592x1944 full resolution full FOV 10-bit mode.
 	 */
 	{
-		{
+		.format = {
 			.code = MEDIA_BUS_FMT_SBGGR10_1X10,
 			.colorspace = V4L2_COLORSPACE_SRGB,
 			.field = V4L2_FIELD_NONE,
 			.width = 2592,
 			.height = 1944
 		},
-		ov5647_2592x1944_10bit,
-		ARRAY_SIZE(ov5647_2592x1944_10bit)
+		.crop = {
+			.left = 0,
+			.top = 0,
+			.width = 2624,
+			.height = 1956,
+		},
```

Comment:
Should this not be 2592x1944?

Reply:
This was the mess I got into over native array vs active vs TIMING_x_OUTPUT_SIZE vs TIMING_x_ADDR_END-TIMING_x_ADDR_START vs what the heck was wanted by userspace.

Yes, 2592x1944 is probably correct.